frontend/script.js

```diff
@@ -0,0 +1,40 @@
+async function makeRequest(postcode) {
+	const response = await fetch(`https://postcode-geocoder.onrender.com/postcode/${postcode}`);
+	const jsonResponse = await response.json();
+	if ("error" in jsonResponse) {
+		const errormsg = jsonResponse["error"];
+		return errormsg;
```

Comment:
perhaps one for later, but you could also shorten this to be return jsonResponse["error"]

There might also be away of getting this whole block into a oneliner.
But take that as an optional improvement

Reply:
Refactored, thanks!